internal/cli/console.go

```diff
@@ -351,14 +353,39 @@ func (o *ConsoleOptions) connectViaWS(ctx context.Context, config *client.Config
 }
 
 func (o *ConsoleOptions) emitUpgradeFailureError(ctx context.Context, name string, origErr error) {
+	// Try to parse error message for HTTP status code and message
+	// Format: "websocket: bad handshake (409 Conflict): Device is decommissioned"
+	errStr := origErr.Error()
+	if strings.Contains(errStr, "bad handshake") {
```

Comment:
if this is a real error can we unwrap it instead string matching? errors.Is(err, ErrBadHandshake)